src/modules/github/action.ts

```diff
@@ -5,18 +5,19 @@ export const SEARCH_GITHUB_REPOSITORIES_REQUEST = 'SEARCH_GITHUB_REPOSITORIES_RE
 export const SEARCH_GITHUB_REPOSITORIES_SUCCESS = 'SEARCH_GITHUB_REPOSITORIES_SUCCESS';
 export const SEARCH_GITHUB_REPOSITORIES_ERROR = 'SEARCH_GITHUB_REPOSITORIES_ERROR';
 
-export const searchGithubRepositories = (name: string) => async (dispatch: Dispatch) => {
+export const searchGithubRepositories = (name: string, perPage: number, currentPage = 1) => async (dispatch: Dispatch) => {
```

Comment:
Simplify arguments naming:
```suggestion
export const searchGithubRepositories = (name: string, size: number, page = 1) => async (dispatch: Dispatch) => {
```